### ISSUE TITLE
More detail for the development build documentation

### ISSUE THREAD

**Issue by jernst** [SSilence/selfoss#1023]
I've been attempting to follow the instructions in "DEVELOPMENT" on https://github.com/SSilence/selfoss to produce a zip from git, and I'm having difficulties to produce a package with the same content as the packaged 2.17 release.

Specifically I'm missing:
* `data/fulltextrss`
* `libs/fulltextrss`
* `public/all-v*.css`
* `public/all-v*.js`

All of those may or may not represent problems... however, if I had to guess, it would appear to me that perhaps I am missing a couple of steps such as for producing the CSS and JS aggregates? Here's my script, based on what that section says:

```
composer install --ignore-platform-reqs
(cd public; npm install)
npm install
grunt install --force # So I don't need to have PHP modules on the build machine
grunt versionupdater compress
```
Is that correct?

(Context: I'm the developer of https://ubos.net/ . UBOS packes Selfoss as one of the apps available for single-command management. Our next UBOS release is ready, except for Selfoss because 2.17 is not compatible with PHP 7.2 and UBOS has moved to 7.2, so I need to find an interim solution until 2.18 is ready.)

**Comment by jtojnar** [comment]
Those files are no longer needed. `data/fulltextrss/standard` is now provided by composer [`j0k3r/graby-site-config`](https://packagist.org/packages/j0k3r/graby-site-config) package is provided by [`j0k3r/graby`](https://packagist.org/packages/j0k3r/graby). Relevant changelog line:

> Fixed Full-text RSS spout (#897)

The `public/all-v*.{css,js}` were renamed to `public/all.{css,js}` but it should not be a part of the release archive – it is generated, and always have been, at run time. Relevant changelog line:

> It is now unlikely that the client browser gets outdated JS or CSS (#907) On Lighttpd, you might need to update your configuration.

---

> Here's my script, based on what that section says

It is slightly redundant – `grunt install` is a shortcut for your first two commands. And `versionupdater` task does nothing unless you provide a `newversion` option – it is used for bumping the selfoss version in the code base. It should work though.

Alternately, you can just use the [pre-built package](https://github.com/SSilence/selfoss#download) of the commit you want. The “development” builds are compiled the same way release tarballs are.

**Comment by jernst** [comment]
Ah! I did not see the packaged development releases (few projects do that, so I didn't specifically look!) Thank you!
